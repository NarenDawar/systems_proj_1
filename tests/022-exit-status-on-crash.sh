#
# Test that minibash correctly sets the $? variable to the exit code of the last program
# even if it terminates with a signal.
# Assumes die is in PATH
#

die -segfault 
echo $?
# 139

die -abort
echo $?
# 134

die -divzero
echo $?
# 136
