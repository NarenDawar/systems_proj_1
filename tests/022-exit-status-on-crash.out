139
134
136
